src/index.html

```diff
@@ -7,12 +7,20 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <base href="/">
     <title>AngularJS application</title>
-    <link rel="stylesheet" href="main.css">
-    <script src="main.js" async></script>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css">
```

Comment:
If you install bootstrap as a dependency you can use it in the template to include the result css instead of using cdn

Reply:
Done.